doc/schema.md

```diff
@@ -41,6 +41,12 @@ When the indexer is synced up to the tip of the chain, the hash of the tip is sa
 
  * `"t" →  "{blockhash}"`
 
+ Each MimbleWimble output results in following new row:
+
+ * `"MWO{output-id}" → "{serialized-output}"`
+
+ Spent outputs are then removed, so only usnpent outputs are left in DB.
```

Comment:
usnpent => unspent

Why remove the spent rows? How do you deal with reorgs?

Reply:
I do it to not waste time and memory on spent outputs, because they are not useful to the client: they can't be spent and don't contribute to available balance.
Am I missing something?

Reply:
Fixed typo.